src/settings-ui/Microsoft.PowerToys.Settings.UI/loc/nl/src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw.lcl

```diff
@@ -856,11 +856,20 @@
         </Str>
         <Disp Icon="Str" />
       </Item>
-      <Item ItemId=";FancyZones_SpanZonesAcrossMonitorsCheckBoxControl.Content" ItemType="0;.resx" PsrId="211" Leaf="true">
+      <Item ItemId=";FancyZones_SpanZonesAcrossMonitorsCheckBoxControl.Text" ItemType="0;.resx" PsrId="211" Leaf="true">
         <Str Cat="Text">
-          <Val><![CDATA[Allow zones to span across monitors (all monitors must have the same DPI scaling)]]></Val>
+          <Val><![CDATA[Allow zones to span across monitors]]></Val>
           <Tgt Cat="Text" Stat="Loc" Orig="New">
-            <Val><![CDATA[Zones over meerdere schermen toestaan (alle schermen moeten dezelfde DPI-schaal hebben)]]></Val>
+            <Val><![CDATA[Toestaan dat zones worden overspannen tussen monitors]]></Val>
```

Comment:
The word "overspannen" _might_ be plausible, but is normally **never** used in this context. Please revise.

Reply:
Thank you for the feedback. Could you please create an issue for this and we'll bring it up with the loc team. Currently it's an external tool that creates these.